### PR TITLE
added a note on primary email

### PR DESCRIPTION
**Pull request by himeshsiriwardana**
## Purpose
$subject

image

### CODE REVIEW

**Review by KaveeshaPiumini — APPROVED** [review]

File: en/identity-server/7.0.0/docs/apis/restapis/scim2-users.yaml [hunk 1]
```diff
@@ -523,6 +531,54 @@ components:
...
+            manager:
+              type: object
+              properties:
+                value:
+                  type: string
+                  example: Taylor
+        urn:scim:wso2:schema:
+          type: object
+          properties:
+            askPassword:
```
Comment (AnuradhaSK) [inline en/identity-server/7.0.0/docs/apis/restapis/scim2-users.yaml:578]: This is not under `urn:scim:wso2:schema` schema in IS.
It should fall under `urn:ietf:params:scim:schemas:extension:enterprise:2.0:User` for IS
for Asgardeo, askPassword is under `urn:scim:wso2:schema`

**Comment by AnuradhaSK** [comment]
Need to update in asgadreo folder's API as well.
Also, organization level SCIM user APIs  under 7.0.0/next and Asgardeo

File: en/identity-server/next/docs/apis/restapis/scim2-users.yaml [hunk 2]
```diff
@@ -523,6 +531,54 @@ components:
...
+              properties:
+                value:
+                  type: string
+                  example: Taylor
+        urn:scim:wso2:schema:
+          type: object
+          properties:
+            askPassword:
+              type: boolean
+              description: If this parameter is set to `true`, users will be allowed configure their own password. An email is sent to the specified email address, which will have instructions for the user to set the password and confirm the new user account.
```
Comment (AnuradhaSK) [inline en/identity-server/next/docs/apis/restapis/scim2-users.yaml:580]: https://github.com/wso2/docs-is/pull/4891#discussion_r1862151162

File: en/identity-server/7.0.0/docs/apis/restapis/scim2-users.yaml (hunk 1 above)
Comment (himeshsiriwardana) [reply en/identity-server/7.0.0/docs/apis/restapis/scim2-users.yaml]: fixed with: https://github.com/wso2/docs-is/pull/4891/commits/c12a75e0702c62e0becb417c7b5e989dcad5cd49

File: en/identity-server/next/docs/apis/restapis/scim2-users.yaml (hunk 2 above)
Comment (himeshsiriwardana) [reply en/identity-server/next/docs/apis/restapis/scim2-users.yaml]: fixed with: https://github.com/wso2/docs-is/pull/4891/commits/c12a75e0702c62e0becb417c7b5e989dcad5cd49

**Comment by himeshsiriwardana** [comment]
These changes will be committed with this PR: https://github.com/wso2/docs-is/pull/4900

**Pull request closed without merging**